pkg/crd/cloudprivateipconfig/v1/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package v1 contains API Schema definitions for the cloud v1 API group
```

Comment:
should this live here or in `openshift/api`? reference: there was a convo on slack around crds should live in api repo.

Reply:
Oh, could you link to the slack thread where the discussion was happening?